lux/js/app.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+require( 'es5-shim' );
+require( 'es5-shim/es5-sham' );
+require( 'babel/polyfill' );
```

Comment:
@ifandelse please could you remove all spaces from your function call statements, see [contributing guidelines](https://github.com/voronianski/flux-comparison/blob/master/CONTRIBUTING.md#there-are-not-too-much-of-rules-to-follow).